docs/source/examples/Layout Example.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipyleaflet import Map, basemaps, basemap_to_tiles, Heatmap, TileLayer\n",
```

Comment:
Maybe adding a markdown cell explaining that this is a demo notebook featuring the applayout, which is detailed in notebook foobar.ipynb

Also specifying that this notebook requires `ipyleaflet` and `bqplot`.

Reply:
Done!